Group-IB hot fix integration

LanskoyGIB opened the pull request:
<!-- REMINDER: THIS IS A PUBLIC REPO DO NOT POST HERE SECRETS/SENSITIVE DATA -->
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/demisto/content?pullRequest=30470) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/demisto/content?pullRequest=30470) before we can accept your contribution.<br/>**13** out of **16** committers have signed the CLA.<br/><br/>:white_check_mark: content-bot<br/>:white_check_mark: daryakoval<br/>:white_check_mark: samuelFain<br/>:white_check_mark: ilappe<br/>:white_check_mark: tkatzir<br/>:white_check_mark: efelmandar<br/>:white_check_mark: ilaner<br/>:white_check_mark: TalNos<br/>:white_check_mark: mmhw<br/>:white_check_mark: ShahafBenYakir<br/>:white_check_mark: maimorag<br/>:white_check_mark: MosheEichler<br/>:white_check_mark: LanskoyGIB<br/>:x: nkanon<br/>:x: kobymeir<br/>:x: ArikDay<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/demisto/content?pullRequest=30470) it.</sub>

MosheEichler commented on Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIBTIA/GroupIBTIA.yml:
Please run the [demisto-sdk format](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/format/README.md) command on this file

MosheEichler commented on Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIBTIA/GroupIBTIA.py:
```diff
@@ -1,8 +1,3 @@
-
-import demistomock as demisto
-from CommonServerPython import *
-from CommonServerUserPython import *
```
Please revert

MosheEichler commented on Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIBTIA/GroupIBTIA.py:
```diff
@@ -1032,16 +1030,13 @@ def fetch_incidents_command(client: Client, last_run: dict, first_fetch_time: st
         portions = client.create_poll_generator(collection_name=collection_name, max_requests=requests_count,
                                                 last_fetch=last_fetch, first_fetch_time=first_fetch_time)
         for portion, last_fetch in portions:
-            last_test = last_fetch
-            for last in last_test:
-                set(last)
```
Why did you remove it?

MosheEichler commented on Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIBTIA/GroupIBTIA.py:
```diff
@@ -1086,7 +1081,7 @@ def fetch_incidents_command(client: Client, last_run: dict, first_fetch_time: st
 
 
-def get_available_collections_command(client: Client):
+def get_available_collections_command(client: Client, args):
```
Why did you add the args parameter?

MosheEichler commented on Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIB_TIA_Feed/GroupIB_TIA_Feed.py:
```diff
@@ -1,9 +1,3 @@
-import requests
-
-import demistomock as demisto
-import demistomock
-from CommonServerPython import *
-from CommonServerUserPython import *
```
Please revert

MosheEichler reviewed (commented):
Hi @LanskoyGIB,
Thank you for your contribution!
Good work :)

Plesae see my comments.

Please feel free to reach out to me with any questions - I'm available here or on slack :)
Thanks again

melamedbn commented:
Hi @LanskoyGIB,

Everything looks great except for deleting the layout, which might impact users who use it.
Why did you delete it instead of re-branding it like the other content items?

Best regards,
Ben

LanskoyGIB commented:
Hello,

This layout was related to deprecated endpoint bp/domain, we removed from
our integration.

Does it can disrupt integration if it was removed with related end-point?

Regards,
Daniil

On Sun, Oct 29, 2023 at 2:55 PM Ben Melamed ***@***.***>
wrote:

> Hi @LanskoyGIB <https://github.com/LanskoyGIB>,
>
> Everything looks great except for deleting the layout, which might impact
> users who use it.
> Why did you delete it instead of re-branding it like the other content
> items?
>
> Best regards,
> Ben
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/demisto/content/pull/30470#issuecomment-1784082432>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZXPBJUABFZJIGXI2SH33V3YBYYZ5AVCNFSM6AAAAAA6RIXQICVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOBUGA4DENBTGI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
<https://www.group-ib.com/> <https://www.group-ib.com/>

DANIIL Lanskoy

Developer | Group-IB

Facebook <https://www.facebook.com/groupibHQ>   Twitter
<https://twitter.com/GroupIB_GIB>   LinkedIn
<https://linkedin.com/company/group-ib?trk=top_nav_home>   Telegram
<https://t.me/groupibhq>


STRICTLY CONFIDENTIAL

This message contains information constituting a trade secret that cannot
be disclosed or distributed without the prior consent of the sender or an
authorized representative of Group-IB. If you have received this message in
error, please notify the sender immediately, delete the message, and do not
disclose the contents to anyone.

LanskoyGIB commented:
Hello again,

Is it possible to schedule a call with you, to except all misunderstanding
on Monday 14pm gmt+3?

Regards,
Daniil

On Mon, Oct 30, 2023 at 3:59 AM Daniil Lanskoy ***@***.***>
wrote:

> Hello,
>
> This layout was related to deprecated endpoint bp/domain, we removed from
> our integration.
>
> Does it can disrupt integration if it was removed with related end-point?
>
> Regards,
> Daniil
>
> On Sun, Oct 29, 2023 at 2:55 PM Ben Melamed ***@***.***>
> wrote:
>
>> Hi @LanskoyGIB <https://github.com/LanskoyGIB>,
>>
>> Everything looks great except for deleting the layout, which might impact
>> users who use it.
>> Why did you delete it instead of re-branding it like the other content
>> items?
>>
>> Best regards,
>> Ben
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/demisto/content/pull/30470#issuecomment-1784082432>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AZXPBJUABFZJIGXI2SH33V3YBYYZ5AVCNFSM6AAAAAA6RIXQICVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOBUGA4DENBTGI>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>
>
> --
> <https://www.group-ib.com/> <https://www.group-ib.com/>
>
> DANIIL Lanskoy
>
> Developer | Group-IB
>
> Facebook <https://www.facebook.com/groupibHQ>   Twitter
> <https://twitter.com/GroupIB_GIB>   LinkedIn
> <https://linkedin.com/company/group-ib?trk=top_nav_home>   Telegram
> <https://t.me/groupibhq>
>
>
> STRICTLY CONFIDENTIAL
>
> This message contains information constituting a trade secret that cannot
> be disclosed or distributed without the prior consent of the sender or an
> authorized representative of Group-IB. If you have received this message in
> error, please notify the sender immediately, delete the message, and do not
> disclose the contents to anyone.
>


-- 
<https://www.group-ib.com/> <https://www.group-ib.com/>

DANIIL Lanskoy

Developer | Group-IB

Facebook <https://www.facebook.com/groupibHQ>   Twitter
<https://twitter.com/GroupIB_GIB>   LinkedIn
<https://linkedin.com/company/group-ib?trk=top_nav_home>   Telegram
<https://t.me/groupibhq>


STRICTLY CONFIDENTIAL

This message contains information constituting a trade secret that cannot
be disclosed or distributed without the prior consent of the sender or an
authorized representative of Group-IB. If you have received this message in
error, please notify the sender immediately, delete the message, and do not
disclose the contents to anyone.

LanskoyGIB commented:
Hello,

We will add this information about this to the RN, since this is a
deprecated endpoint, which was removed from our API.

Also can we schedule a call before you will publish new release to discuss
all the changes and escape any misunderstandings?

From our side we can provide you testing credentials. It could help you
complete test of new app.

Best regards,
Daniil
On Mon, Oct 30, 2023 at 4:52 PM Ben Melamed ***@***.***>
wrote:

> Hello, This layout was related to deprecated endpoint bp/domain, we
> removed from our integration. Does it can disrupt integration if it was
> removed with related end-point? Regards, Daniil
> …
> <#m_-3279291781457105430_m_-7871358643558869910_m_8315557679875658306_m_-3662237403425344578_>
> On Sun, Oct 29, 2023 at 2:55 PM Ben Melamed *@*.*> wrote: Hi @LanskoyGIB
> <https://github.com/LanskoyGIB> https://github.com/LanskoyGIB
> <https://github.com/LanskoyGIB>, Everything looks great except for deleting
> the layout, which might impact users who use it. Why did you delete it
> instead of re-branding it like the other content items? Best regards, Ben —
> Reply to this email directly, view it on GitHub <#30470 (comment)
> <https://github.com/demisto/content/pull/30470#issuecomment-1784082432>>,
> or unsubscribe
> https://github.com/notifications/unsubscribe-auth/AZXPBJUABFZJIGXI2SH33V3YBYYZ5AVCNFSM6AAAAAA6RIXQICVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOBUGA4DENBTGI
> <https://github.com/notifications/unsubscribe-auth/AZXPBJUABFZJIGXI2SH33V3YBYYZ5AVCNFSM6AAAAAA6RIXQICVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOBUGA4DENBTGI>
> . You are receiving this because you were mentioned.Message ID: @.*>
> -- https://www.group-ib.com/ https://www.group-ib.com/ DANIIL Lanskoy
> Developer | Group-IB Facebook https://www.facebook.com/groupibHQ Twitter
> https://twitter.com/GroupIB_GIB LinkedIn
> https://linkedin.com/company/group-ib?trk=top_nav_home Telegram
> https://t.me/groupibhq STRICTLY CONFIDENTIAL This message contains
> information constituting a trade secret that cannot be disclosed or
> distributed without the prior consent of the sender or an authorized
> representative of Group-IB. If you have received this message in error,
> please notify the sender immediately, delete the message, and do not
> disclose the contents to anyone.
>
> It could impact existing alerts that use the layout, you have two options:
>
>    1. Breaking Changes announcement and RN
>    2. Keep the layout and fix it in this PR or as part of another effort.
>
> Let me know what you think.
>
> Best regards,
> Ben
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/demisto/content/pull/30470#issuecomment-1785248610>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZXPBJUCQACNPEIEG6LF2YLYB6WJXAVCNFSM6AAAAAA6RIXQICVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOBVGI2DQNRRGA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
<https://www.group-ib.com/> <https://www.group-ib.com/>

DANIIL Lanskoy

Developer | Group-IB

Facebook <https://www.facebook.com/groupibHQ>   Twitter
<https://twitter.com/GroupIB_GIB>   LinkedIn
<https://linkedin.com/company/group-ib?trk=top_nav_home>   Telegram
<https://t.me/groupibhq>


STRICTLY CONFIDENTIAL

This message contains information constituting a trade secret that cannot
be disclosed or distributed without the prior consent of the sender or an
authorized representative of Group-IB. If you have received this message in
error, please notify the sender immediately, delete the message, and do not
disclose the contents to anyone.


-- 
<https://www.group-ib.com/> <https://www.group-ib.com/>

DANIIL Lanskoy

Developer | Group-IB

Facebook <https://www.facebook.com/groupibHQ>   Twitter
<https://twitter.com/GroupIB_GIB>   LinkedIn
<https://linkedin.com/company/group-ib?trk=top_nav_home>   Telegram
<https://t.me/groupibhq>


STRICTLY CONFIDENTIAL

This message contains information constituting a trade secret that cannot
be disclosed or distributed without the prior consent of the sender or an
authorized representative of Group-IB. If you have received this message in
error, please notify the sender immediately, delete the message, and do not
disclose the contents to anyone.

melamedbn commented:
> Hello, We will add this information about this to the RN, since this is a deprecated endpoint, which was removed from our API. Also can we schedule a call before you will publish new release to discuss all the changes and escape any misunderstandings? From our side we can provide you testing credentials. It could help you complete test of new app. Best regards, Daniil On Mon, Oct 30, 2023 at 4:52 PM Ben Melamed ***@***.***> wrote:
> […](#)
> Hello, This layout was related to deprecated endpoint bp/domain, we removed from our integration. Does it can disrupt integration if it was removed with related end-point? Regards, Daniil … <#m_-3279291781457105430_m_-7871358643558869910_m_8315557679875658306_m_-3662237403425344578_> On Sun, Oct 29, 2023 at 2:55 PM Ben Melamed *@*.*> wrote: Hi @LanskoyGIB <https://github.com/LanskoyGIB> https://github.com/LanskoyGIB <https://github.com/LanskoyGIB>, Everything looks great except for deleting the layout, which might impact users who use it. Why did you delete it instead of re-branding it like the other content items? Best regards, Ben — Reply to this email directly, view it on GitHub <#30470 (comment) <[#30470 (comment)](https://github.com/demisto/content/pull/30470#issuecomment-1784082432)>>, or unsubscribe https://github.com/notifications/unsubscribe-auth/AZXPBJUABFZJIGXI2SH33V3YBYYZ5AVCNFSM6AAAAAA6RIXQICVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOBUGA4DENBTGI <https://github.com/notifications/unsubscribe-auth/AZXPBJUABFZJIGXI2SH33V3YBYYZ5AVCNFSM6AAAAAA6RIXQICVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOBUGA4DENBTGI> . You are receiving this because you were mentioned.Message ID: @.*> -- https://www.group-ib.com/ https://www.group-ib.com/ DANIIL Lanskoy Developer | Group-IB Facebook https://www.facebook.com/groupibHQ Twitter https://twitter.com/GroupIB_GIB LinkedIn https://linkedin.com/company/group-ib?trk=top_nav_home Telegram https://t.me/groupibhq STRICTLY CONFIDENTIAL This message contains information constituting a trade secret that cannot be disclosed or distributed without the prior consent of the sender or an authorized representative of Group-IB. If you have received this message in error, please notify the sender immediately, delete the message, and do not disclose the contents to anyone. It could impact existing alerts that use the layout, you have two options: 1. Breaking Changes announcement and RN 2. Keep the layout and fix it in this PR or as part of another effort. Let me know what you think. Best regards, Ben — Reply to this email directly, view it on GitHub <[#30470 (comment)](https://github.com/demisto/content/pull/30470#issuecomment-1785248610)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AZXPBJUCQACNPEIEG6LF2YLYB6WJXAVCNFSM6AAAAAA6RIXQICVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOBVGI2DQNRRGA> . You are receiving this because you were mentioned.Message ID: ***@***.***>
> -- <https://www.group-ib.com/> <https://www.group-ib.com/> DANIIL Lanskoy Developer | Group-IB Facebook <https://www.facebook.com/groupibHQ> Twitter <https://twitter.com/GroupIB_GIB> LinkedIn <https://linkedin.com/company/group-ib?trk=top_nav_home> Telegram <https://t.me/groupibhq> STRICTLY CONFIDENTIAL This message contains information constituting a trade secret that cannot be disclosed or distributed without the prior consent of the sender or an authorized representative of Group-IB. If you have received this message in error, please notify the sender immediately, delete the message, and do not disclose the contents to anyone.
> -- <https://www.group-ib.com/> <https://www.group-ib.com/> DANIIL Lanskoy Developer | Group-IB Facebook <https://www.facebook.com/groupibHQ> Twitter <https://twitter.com/GroupIB_GIB> LinkedIn <https://linkedin.com/company/group-ib?trk=top_nav_home> Telegram <https://t.me/groupibhq> STRICTLY CONFIDENTIAL This message contains information constituting a trade secret that cannot be disclosed or distributed without the prior consent of the sender or an authorized representative of Group-IB. If you have received this message in error, please notify the sender immediately, delete the message, and do not disclose the contents to anyone.

In that case, what about the GIB Brand Protection Domain incident type? do you need it?
If it stays, let's change the default layout for that incident type.
What about all the other content items that weren't rebranded? 

best regards,
Ben

LanskoyGIB commented:
Hello,

We removed bp/domain incident type.

What kind contents you mean? Because when we prepared last PR, PA team
explained us that we are not able to change all the the fields with name
(for example: package name, integrations name, any fields inside yaml file)
 and we changed only the fields that PA team told we could.

Regards,
Daniil

On Wed, Nov 1, 2023 at 2:00 PM Ben Melamed ***@***.***> wrote:

> Hello, We will add this information about this to the RN, since this is a
> deprecated endpoint, which was removed from our API. Also can we schedule a
> call before you will publish new release to discuss all the changes and
> escape any misunderstandings? From our side we can provide you testing
> credentials. It could help you complete test of new app. Best regards,
> Daniil On Mon, Oct 30, 2023 at 4:52 PM Ben Melamed *@*.
>
> *> wrote: … <#m_-7892626226964915101_> Hello, This layout was related to
> deprecated endpoint bp/domain, we removed from our integration. Does it can
> disrupt integration if it was removed with related end-point? Regards,
> Daniil …
> <#m_-3279291781457105430_m_-7871358643558869910_m_8315557679875658306_m_-3662237403425344578_>
> On Sun, Oct 29, 2023 at 2:55 PM Ben Melamed @.> wrote: Hi @LanskoyGIB
> <https://github.com/LanskoyGIB> https://github.com/LanskoyGIB
> <https://github.com/LanskoyGIB> https://github.com/LanskoyGIB
> <https://github.com/LanskoyGIB> https://github.com/LanskoyGIB
> <https://github.com/LanskoyGIB>, Everything looks great except for deleting
> the layout, which might impact users who use it. Why did you delete it
> instead of re-branding it like the other content items? Best regards, Ben —
> Reply to this email directly, view it on GitHub <#30470
> <https://github.com/demisto/content/pull/30470> (comment) <#30470 (comment)
> <https://github.com/demisto/content/pull/30470#issuecomment-1784082432>>>,
> or unsubscribe
> https://github.com/notifications/unsubscribe-auth/AZXPBJUABFZJIGXI2SH33V3YBYYZ5AVCNFSM6AAAAAA6RIXQICVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOBUGA4DENBTGI
> <https://github.com/notifications/unsubscribe-auth/AZXPBJUABFZJIGXI2SH33V3YBYYZ5AVCNFSM6AAAAAA6RIXQICVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOBUGA4DENBTGI>
> https://github.com/notifications/unsubscribe-auth/AZXPBJUABFZJIGXI2SH33V3YBYYZ5AVCNFSM6AAAAAA6RIXQICVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOBUGA4DENBTGI
> <https://github.com/notifications/unsubscribe-auth/AZXPBJUABFZJIGXI2SH33V3YBYYZ5AVCNFSM6AAAAAA6RIXQICVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOBUGA4DENBTGI>
> . You are receiving this because you were mentioned.Message ID: @.> --
> https://www.group-ib.com/ <https://www.group-ib.com/>
> https://www.group-ib.com/ <https://www.group-ib.com/> DANIIL Lanskoy
> Developer | Group-IB Facebook https://www.facebook.com/groupibHQ
> <https://www.facebook.com/groupibHQ> Twitter
> https://twitter.com/GroupIB_GIB <https://twitter.com/GroupIB_GIB> LinkedIn
> https://linkedin.com/company/group-ib?trk=top_nav_home
> <https://linkedin.com/company/group-ib?trk=top_nav_home> Telegram
> https://t.me/groupibhq <https://t.me/groupibhq> STRICTLY CONFIDENTIAL This
> message contains information constituting a trade secret that cannot be
> disclosed or distributed without the prior consent of the sender or an
> authorized representative of Group-IB. If you have received this message in
> error, please notify the sender immediately, delete the message, and do not
> disclose the contents to anyone. It could impact existing alerts that use
> the layout, you have two options: 1. Breaking Changes announcement and RN
> 2. Keep the layout and fix it in this PR or as part of another effort. Let
> me know what you think. Best regards, Ben — Reply to this email directly,
> view it on GitHub <#30470 (comment)
> <https://github.com/demisto/content/pull/30470#issuecomment-1785248610>>,
> or unsubscribe
> https://github.com/notifications/unsubscribe-auth/AZXPBJUCQACNPEIEG6LF2YLYB6WJXAVCNFSM6AAAAAA6RIXQICVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOBVGI2DQNRRGA
> <https://github.com/notifications/unsubscribe-auth/AZXPBJUCQACNPEIEG6LF2YLYB6WJXAVCNFSM6AAAAAA6RIXQICVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOBVGI2DQNRRGA>
> . You are receiving this because you were mentioned.Message ID: @.***>
> -- https://www.group-ib.com/ https://www.group-ib.com/ DANIIL Lanskoy
> Developer | Group-IB Facebook https://www.facebook.com/groupibHQ Twitter
> https://twitter.com/GroupIB_GIB LinkedIn
> https://linkedin.com/company/group-ib?trk=top_nav_home Telegram
> https://t.me/groupibhq STRICTLY CONFIDENTIAL This message contains
> information constituting a trade secret that cannot be disclosed or
> distributed without the prior consent of the sender or an authorized
> representative of Group-IB. If you have received this message in error,
> please notify the sender immediately, delete the message, and do not
> disclose the contents to anyone.
> -- https://www.group-ib.com/ https://www.group-ib.com/ DANIIL Lanskoy
> Developer | Group-IB Facebook https://www.facebook.com/groupibHQ Twitter
> https://twitter.com/GroupIB_GIB LinkedIn
> https://linkedin.com/company/group-ib?trk=top_nav_home Telegram
> https://t.me/groupibhq STRICTLY CONFIDENTIAL This message contains
> information constituting a trade secret that cannot be disclosed or
> distributed without the prior consent of the sender or an authorized
> representative of Group-IB. If you have received this message in error,
> please notify the sender immediately, delete the message, and do not
> disclose the contents to anyone.
>
> In that case, what about the GIB Brand Protection Domain incident type? do
> you need it?
> If it stays, let's change the default layout for that incident type.
> What about all the other content items that weren't rebranded?
>
> best regards,
> Ben
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/demisto/content/pull/30470#issuecomment-1788768179>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZXPBJRYLWKZDD7L3WP6LHTYCITTNAVCNFSM6AAAAAA6RIXQICVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOBYG43DQMJXHE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
<https://www.group-ib.com/> <https://www.group-ib.com/>

DANIIL Lanskoy

Developer | Group-IB

Facebook <https://www.facebook.com/groupibHQ>   Twitter
<https://twitter.com/GroupIB_GIB>   LinkedIn
<https://linkedin.com/company/group-ib?trk=top_nav_home>   Telegram
<https://t.me/groupibhq>


STRICTLY CONFIDENTIAL

This message contains information constituting a trade secret that cannot
be disclosed or distributed without the prior consent of the sender or an
authorized representative of Group-IB. If you have received this message in
error, please notify the sender immediately, delete the message, and do not
disclose the contents to anyone.

MosheEichler commented:
@LanskoyGIB Please fill in the [contribution form](https://forms.gle/XDfxU4E61ZwEESSMA)

LanskoyGIB commented:
Hello,

Done with fill the form.

Regards,
Daniil

On Sun, Nov 5, 2023 at 12:14 PM Moshe Eichler ***@***.***>
wrote:

> @LanskoyGIB <https://github.com/LanskoyGIB> Please fill in the contribution
> form <https://forms.gle/XDfxU4E61ZwEESSMA>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/demisto/content/pull/30470#issuecomment-1793681007>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZXPBJWUNRBO3XSPLFHG2RTYC5KFZAVCNFSM6AAAAAA6RIXQICVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOJTGY4DCMBQG4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
<https://www.group-ib.com/> <https://www.group-ib.com/>

DANIIL Lanskoy

Developer | Group-IB

Facebook <https://www.facebook.com/groupibHQ>   Twitter
<https://twitter.com/GroupIB_GIB>   LinkedIn
<https://linkedin.com/company/group-ib?trk=top_nav_home>   Telegram
<https://t.me/groupibhq>


STRICTLY CONFIDENTIAL

This message contains information constituting a trade secret that cannot
be disclosed or distributed without the prior consent of the sender or an
authorized representative of Group-IB. If you have received this message in
error, please notify the sender immediately, delete the message, and do not
disclose the contents to anyone.

MosheEichler commented:
> Hello, Done with fill the form. Regards, Daniil
> […](#)
> On Sun, Nov 5, 2023 at 12:14 PM Moshe Eichler ***@***.***> wrote: @LanskoyGIB <https://github.com/LanskoyGIB> Please fill in the contribution form <https://forms.gle/XDfxU4E61ZwEESSMA> — Reply to this email directly, view it on GitHub <[#30470 (comment)](https://github.com/demisto/content/pull/30470#issuecomment-1793681007)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AZXPBJWUNRBO3XSPLFHG2RTYC5KFZAVCNFSM6AAAAAA6RIXQICVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOJTGY4DCMBQG4> . You are receiving this because you were mentioned.Message ID: ***@***.***>
> -- <https://www.group-ib.com/> <https://www.group-ib.com/> DANIIL Lanskoy Developer | Group-IB Facebook <https://www.facebook.com/groupibHQ> Twitter <https://twitter.com/GroupIB_GIB> LinkedIn <https://linkedin.com/company/group-ib?trk=top_nav_home> Telegram <https://t.me/groupibhq> STRICTLY CONFIDENTIAL This message contains information constituting a trade secret that cannot be disclosed or distributed without the prior consent of the sender or an authorized representative of Group-IB. If you have received this message in error, please notify the sender immediately, delete the message, and do not disclose the contents to anyone.

It looks like the form isn't filled out for some reason
Let's try to do it together
Send me a message in Slack [here](https://start.paloaltonetworks.com/join-our-slack-community), look for me as Moshe Eichler

LanskoyGIB commented:
Hello,

For some reason i can not join.

[image: image.png]

Regards,
Daniil

On Tue, Nov 7, 2023 at 2:16 PM Moshe Eichler ***@***.***>
wrote:

> Hello, Done with fill the form. Regards, Daniil
> … <#m_9072837539928308233_>
> On Sun, Nov 5, 2023 at 12:14 PM Moshe Eichler *@*.*> wrote: @LanskoyGIB
> <https://github.com/LanskoyGIB> https://github.com/LanskoyGIB
> <https://github.com/LanskoyGIB> Please fill in the contribution form
> https://forms.gle/XDfxU4E61ZwEESSMA <https://forms.gle/XDfxU4E61ZwEESSMA> —
> Reply to this email directly, view it on GitHub <#30470 (comment)
> <https://github.com/demisto/content/pull/30470#issuecomment-1793681007>>,
> or unsubscribe
> https://github.com/notifications/unsubscribe-auth/AZXPBJWUNRBO3XSPLFHG2RTYC5KFZAVCNFSM6AAAAAA6RIXQICVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOJTGY4DCMBQG4
> <https://github.com/notifications/unsubscribe-auth/AZXPBJWUNRBO3XSPLFHG2RTYC5KFZAVCNFSM6AAAAAA6RIXQICVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOJTGY4DCMBQG4>
> . You are receiving this because you were mentioned.Message ID: @.*>
> -- https://www.group-ib.com/ https://www.group-ib.com/ DANIIL Lanskoy
> Developer | Group-IB Facebook https://www.facebook.com/groupibHQ Twitter
> https://twitter.com/GroupIB_GIB LinkedIn
> https://linkedin.com/company/group-ib?trk=top_nav_home Telegram
> https://t.me/groupibhq STRICTLY CONFIDENTIAL This message contains
> information constituting a trade secret that cannot be disclosed or
> distributed without the prior consent of the sender or an authorized
> representative of Group-IB. If you have received this message in error,
> please notify the sender immediately, delete the message, and do not
> disclose the contents to anyone.
>
> It looks like the form isn't filled out for some reason
> Let's try to do it together
> Send me a message in Slack here
> <https://start.paloaltonetworks.com/join-our-slack-community>, look for
> me as Moshe Eichler
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/demisto/content/pull/30470#issuecomment-1798304815>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZXPBJSQ2OQUAK4T7FW5EKLYDIKCHAVCNFSM6AAAAAA6RIXQICVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOJYGMYDIOBRGU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
<https://www.group-ib.com/> <https://www.group-ib.com/>

DANIIL Lanskoy

Developer | Group-IB

Facebook <https://www.facebook.com/groupibHQ>   Twitter
<https://twitter.com/GroupIB_GIB>   LinkedIn
<https://linkedin.com/company/group-ib?trk=top_nav_home>   Telegram
<https://t.me/groupibhq>


STRICTLY CONFIDENTIAL

This message contains information constituting a trade secret that cannot
be disclosed or distributed without the prior consent of the sender or an
authorized representative of Group-IB. If you have received this message in
error, please notify the sender immediately, delete the message, and do not
disclose the contents to anyone.

MosheEichler commented:
Hey @LanskoyGIB

Please choose form here a time for a zoon session
https://calendar.app.google/CM8DrUQp4P9g74fs5

Regards,
Moshe
